nondex-common/src/main/java/edu/illinois/nondex/common/Configuration.java

```diff
@@ -178,7 +186,7 @@ public static Configuration parseArgs(Properties props) {
 
         final String testName = props.getProperty("test", null);
 
-        return new Configuration(nonDetKind, seed, filter, start, end, nondexDir, nondexJarDir, testName,
+        return new Configuration(nonDetKind, seed, filter, start, end, log, nondexDir, nondexJarDir, testName,
```

Comment:
Add a method here getSystematicLogPath, that you write as the getFailuresPath is written, and use on all your accesses to the location of the log that method. That will ensure that the log file is created in the .nondex directory rather than in the project home somewhere.